201 - Kubernetes Deep Dive /5. Configuring Kube Apps - Part II Secrets/kubernetes/secret.yaml

```diff
@@ -1,7 +1,6 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: api-key-secret
-type: Opaque
+  name: license-config
 data:
-  api_key.json: YOUR_BASE64_ENCODED_API_KEY
+  license.json: ewogICJsaWNlbnNlX2tleSI6ICJzdXBlciBzZWNyZXQgbGljZW5zZSBrZXkiCn0K
```

Comment:
If you use the `stringData` field in the secret you can avoid explaining the base64 encoding.

I think `stringData` is preferable except when the value can't be represented and base64 encoding is required.

Reply:
Yep I explain that in the video